app/main.py

```diff
@@ -1,3 +1,13 @@
 def cache(func):
-    # Write your code here
-    pass
+    cached_results = {}
+
+    def wrapper(*args):
```

Comment:
You could consider using wraps from functools to ensure that the decorated function maintains its original name and docstring.